_includes/guides.html

```diff
@@ -22,7 +22,7 @@ <h5 class="grid_content grid_content_dark">Overview</h5>
                         <a href="https://rackhd.readthedocs.io/en/latest/tutorials/index.html" class="list-group-item list-group-item-fixed-border">
                              <h5 class="grid_content grid_content_dark">Tutorials</h5>
                         </a>
-                        <a href="https://rackhd.atlassian.net/wiki/display/RAC1/RackHD+Installation+Guide+-1.0.0" class="list-group-item list-group-item-fixed-border">
+                        <a href="https://rackhd.atlassian.net/wiki/display/RAC1/RackHD+Release+Installation+Guide" class="list-group-item list-group-item-fixed-border">
```

Comment:
why not 'Get RackHD' redirect  https://rackhd.atlassian.net/wiki/display/RAC1/RackHD+Release+Page than 'installation guide' page?  or another better page is also fine, because installation guide is only one part of RackHD, and by the way, it's not easy to find 1.1.0 release in 'installation guide' page.